openfe/protocols/openmm_afe/base.py

```diff
@@ -335,9 +335,9 @@ def _get_modeller(
                 mol.assign_partial_charges('am1bcc',
                                            use_conformers=mol.conformers)
 
-            system_generator.create_system(
-                mol.to_topology().to_openmm(), molecules=[mol]
-            )
+        system_generator.create_system(
+            mol.to_topology().to_openmm(), molecules=[mol]
+        )
```

Comment:
@richardjgowers this is the identation bug I mentioned earlier - I should have noticed 🤦🏽‍♂️

Reply:
no this won't work... one sec